modules/architecture-machine-roles.adoc

```diff
@@ -44,12 +44,19 @@ control the Kubernetes cluster. In {product-title}, the master machines are
 the control plane. They contain more
 than just the Kubernetes services for managing the {product-title} cluster.
 Because all of the machines with the control plane role are master machines,
-and the terms "master" and "control plane" are used interchangeably to describe
+the terms _master_ and _control plane_ are used interchangeably to describe
 them. Instead of being grouped into a
 MachineSet, master machines are defined by a series of standalone machine API
 resources. Extra controls apply to master machines to prevent you from deleting
 all master machines and breaking your cluster.
 
+[NOTE]
+====
+It is recommended to only use three master nodes. Although you can theoretically
+use any number of master nodes, the number is constrained by etcd quorum due to
```

Comment:
Not sure whether appropriate to link external https://github.com/etcd-io/etcd/blob/master/Documentation/faq.md#why-an-odd-number-of-cluster-members here. If yes, could link it.

Reply:
@xingxingxia we cannot link to Github links in the official docs. Is there any additional info from that link that you think should be included here?

Reply:
Currently short statement is OK